benchmarks/harness.go

```diff
@@ -0,0 +1,364 @@
+package benchmarks
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/evergreen-ci/pail"
+	"github.com/evergreen-ci/pail/testutil"
+	"github.com/evergreen-ci/poplar"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+func buildDir() string {
+	_, file, _, _ := runtime.Caller(0)
+	return filepath.Join(filepath.Dir(filepath.Dir(file)), "build")
```

Comment:
Shouldn't this be one more level up? Or is caller root?

Reply:
`filepath.Dir` gives you the directory of the current file (i.e. `benchmarks/harness.go/..`)

Reply:
Missed the double filepath.Dir! Thanks